### DISCUSSION TITLE
New RLC-811A IPC_560B158MP

### DISCUSSION THREAD

**Discussion by HVR88** [AT0myks/reolink-fw-archive#37] [Missing firmware]
Just purchased 4 of these cameras along with NVR.

HW version is different/newer than the one listed here and firmware also appears newer:

v2.1.0.2144_23042803

**Comment by AT0myks** [comment]
I think you mean v3.1.0.2144_23042803? This must be a new hardware version because it's different from the only one that's listed for this camera.
